image-build-push.sh

```diff
@@ -0,0 +1,36 @@
+# Terminal output colors 
+GREEN='\033[0;32m'
+ENDCOLOR='\033[0m' # No Color
+RED='\033[0;31m'
+
+# Environment Variables 
+PREFER_PODMAN="${PREFER_PODMAN:-0}"
+PUSH="${PUSH:-1}"
+TAG="${TAG:-test}"
+REGISTRY_ORG="${REGISTRY_ORG:-jezhu}"
+
+echo "${GREEN}Environment Varibles ${ENDCOLOR}"
+echo "PREFER_PODMAN= ${PREFER_PODMAN}"
+echo "PUSH= ${PUSH}"
+echo "TAG= ${TAG}" 
+echo "REGISTRY_ORG= ${REGISTRY_ORG}"
+
+# User Input to confirm environment variables before proceeding. 
+read -p "$(echo ${RED}"Do these env variables look right? (Y/N): " ${ENDCOLOR})" confirm && [[ $confirm == [yY] || $confirm == [yY][eE][sS] ]] || exit 1
+
+if [[ -x "$(command -v podman)" && $PREFER_PODMAN == 1 ]]; then
+    OCI_BIN="podman"
+else
+    OCI_BIN="docker"
+fi
+
+BASE_IMAGE="quay.io/${REGISTRY_ORG}/distributed-tracing-console-plugin"
+IMAGE=${BASE_IMAGE}:${TAG}
+
+echo "Building image '${IMAGE}' with ${OCI_BIN}"
+$OCI_BIN build -t $IMAGE .
```

Comment:
Same thing as in build-image.sh, but the target platform has to be amd64 to run in cluster, and putting it here just makes it much easier for m-series mac users to use.


```suggestion
$OCI_BIN build -t $IMAGE --platform=linux/amd64 .
```